tests/models/vit_hybrid/test_modeling_vit_hybrid.py

```diff
@@ -57,6 +57,7 @@ def __init__(
         attention_probs_dropout_prob=0.1,
         type_sequence_label_size=10,
         initializer_range=0.02,
+        backbone_featmap_shape=[1, 1024, 4, 4],
```

Comment:
I guess `4` is from the fact we have `image_size=64` here in the test, right?
Is `1024` necessary, or we can change it to something smaller?

Reply:
Probably you are right here, let me double check

Reply:
Yes we can! Fixed in [f24a507](https://github.com/huggingface/transformers/pull/20679/commits/f24a507a5bf6c401ab2379d3ed73a8e96ef32f50)

Reply:
Smaller values in [0b28767](https://github.com/huggingface/transformers/pull/20679/commits/0b28767e70978ec4076fb1523f241973a743aa43)